skyvern/webeye/actions/handler.py

```diff
@@ -828,9 +820,16 @@ async def normal_select(
     action: actions.SelectOptionAction,
     skyvern_element: SkyvernElement,
 ) -> List[ActionResult]:
+    try:
```

Comment:
I don't think this is a good change. How can the LLM "unselect" an option? Here doing select option on an already selected option would unselect it before, but now that can never be the case right?

Reply:
What do you mean unselect before select? I don't understand.

Reply:
If llm want to unselect sth. He can return the value represented the "unselect" right? What the problem here?